var/spack/repos/builtin/packages/perl-parsetemplate/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PerlParsetemplate(PerlPackage):
+    """Parse::Template - Processor for templates containing Perl expressions."""
+
+    homepage = "https://metacpan.org/pod/Parse::Template"
+    url = "https://cpan.metacpan.org/authors/id/P/PS/PSCUST/ParseTemplate-3.08.tar.gz"
+
```

Comment:
You could add the license here. The docs say 'same as perl', so this would be:
license("Artistic-1.0-Perl OR GPL-1.0-or-later")